crates/api_models/src/payments.rs

```diff
@@ -293,6 +293,53 @@ pub struct PaymentsRequest {
     pub feature_metadata: Option<FeatureMetadata>,
 }
 
+#[derive(
+    Default, Debug, serde::Serialize, Clone, PartialEq, ToSchema, router_derive::PolymorphicSchema,
+)]
+pub struct PaymentAttemptResponse {
+    /// Unique identifier for the attempt
+    pub attempt_id: String,
```

Comment:
the key name will be `reference_id` and not `attempt_id`. It can be sometimes not attempt_id too